components/persistor/lib/knex/update.ts

```diff
@@ -216,7 +216,9 @@ module.exports = function (PersistObjectTemplate) {
         promises.push(this.saveKnexPojo(obj, pojo, isDocumentUpdate ? obj._id : null, txn, logger));
 
         return Promise.all(promises) // update sql saves first
-            .then(Promise.all(remoteUpdates)) // update remote objects second
+            .then(() => {
```

Comment:
this can just be
```javascript
.then(() => Promise.all(remoteUpdates))
```

Reply:
I like the spirit of this! for the sake of time to make the small change but then retest (yes we do have unit tests, but also I did manual testing on the current branch with actual S3 connection), I'm gonna say this is probably OK as is